ckan/lib/plugins.py

```diff
@@ -553,22 +553,7 @@ def check_data_dict(self, data_dict: dict[str, Any]) -> None:
 
     def setup_template_variables(self, context: Context,
                                  data_dict: dict[str, Any]) -> None:
-        c.is_sysadmin = ckan.authz.is_sysadmin(c.user)
-
-        ## This is messy as auths take domain object not data_dict
-        context_group = context.get('group', None)
-        group = context_group or getattr(c, 'group', None)
-        if group:
-            try:
-                if not context_group:
-                    context['group'] = group
-                logic.check_access('group_change_state', context)
```

Comment:
As a reference, the logic for `group_change_state` is now being executed in a validator: https://github.com/ckan/ckan/blob/37f86db98b8e2e8b67155b8ba1e591873b9ba2de/ckan/logic/validators.py#L547-L568